sound/soc/sof/sof-priv.h

```diff
@@ -113,12 +113,22 @@ struct snd_sof_pdata;
  * @phy_addr:		Platform dependent address to be used, if  @use_phy_addr
  *			is true
  * @no_ipc_position:	Disable position update IPC from firmware
+ * @rate:		Sample rate
```

Comment:
'rate' is a bit too open in terms of naming. This could be confused with bitrate, this should be either sample_rate or frame_rate.